### PR TITLE
[JENKINS-48943] - Whitelist java.util.Collections entries required by the plugin

### PR DESCRIPTION
**Pull request by oleg-nenashev**
Fix in https://github.com/jenkinsci/gerrit-trigger-plugin/commit/93a771948044fab506b5666ad6778b9636cd9f2c is not complete.

https://issues.jenkins-ci.org/browse/JENKINS-48943

@reviewbybees @rsandell

### CODE REVIEW

**Comment by oleg-nenashev** [comment]
@rsandell I cannot setup CI which would verify 2.102, but I confirm that the plugin appears to be fixed after this patch. Would be great to ship it

File: Jenkinsfile [hunk 1]
```diff
@@ -1,5 +1,6 @@
 buildPlugin(
     platforms: ['linux'],
+    jenkinsVersions: [null],
```
Comment (jglick) [inline Jenkinsfile]: Why? That is the default.

File: src/main/resources/META-INF/hudson.remoting.ClassFilter [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Remove once https://github.com/jenkinsci/jenkins/pull/3234 is integrated?
```
Comment (jglick) [inline src/main/resources/META-INF/hudson.remoting.ClassFilter]: link to this PR for convenience

**Comment by oleg-nenashev** [comment]
Should be replaced once https://github.com/jenkinsci/jenkins/pull/3234 is merged

File: Jenkinsfile (hunk 1 above)
Comment (oleg-nenashev) [reply Jenkinsfile]: It is a part of my fiasco with Jenkinsfiling tests against 2.102. Maybe we need to update the blogpost, it does not really work in several plugins due to Maven Enforcer. The blogpost should suggest a flag for disabling it

**Pull request merged**